src/init.js

```diff
@@ -27,7 +26,6 @@ import { config } from "@onflow/config";
  * @param {number} [props.pkey] - private key to use for service account in case of collisions
  */
 export const init = async (basePath, props = {}) => {
-  const { port } = props;
   const { pkey = "48a1f554aeebf6bf9fe0d7b5b79d080700b073ee77909973ea0b2f6fbc902" } = props;
```

Comment:
What's this default key for?

Reply:
Not sure, it was there before me... will look into whether that is necessary.

Reply:
I will make another PR for this as there is a bit more to unpack here than just this (docs, etc)